rcongui/src/components/Scoreboard/GamesScore.js

```diff
@@ -104,10 +112,29 @@ const GamesScore = ({ classes }) => {
   const durationToHour = (val) =>
     new Date(val * 1000).toISOString().substr(11, 5);
 
+  let { slug } = useParams();
+
   const lastRefresh = scores.get("snapshot_timestamp")
     ? moment.unix(scores.get("snapshot_timestamp")).format()
     : "N/A";
 
+  const getShareableLink = () => {
+    return window.location.href;
+  };
+  const doSelectMap = (map_id) => {
```

Comment:
We don't even need a event handler. 
We just need to wrap the maps with this component: https://reactrouter.com/web/api/Link